### PR TITLE
Remove Rtree from oq-libs

### PR DESCRIPTION
**Pull request by daniviga**
Upstream: https://github.com/gem/oq-engine/pull/4661

### CODE REVIEW

File: openquake/libs/__init__.py [hunk 1]
```diff
@@ -16,4 +16,4 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
 
-__version__ = '2.2.2'
+__version__ = '2.2.3'
```
Comment (nastasi-oq) [inline openquake/libs/__init__.py]: In this case we have a broken back-compatibility (removing a package) => version could be 3.0.0,
Why do you consider it a bugfix ?

Comment (daniviga) [reply openquake/libs/__init__.py]: That's right

Comment (daniviga) [reply openquake/libs/__init__.py]: What do you think about a `2.3.0`? I would keep `3.0` for a full distribution bump (i.e. when we'll update all the libs, probably with 3.6)

**Comment by daniviga** [comment]
I will keep `rtee` in `oq-libs` for now since we already have the corresponding wheel. We'll remove it with the bump to Python 3.7

**Pull request closed without merging**